spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -253,25 +255,26 @@
             comp2.install_service('SourceDir\\ProgramFilesFolder\\TestID\\TestProduct\\somedir\\oneUpAgain\\twoUp\\bin.exe')
             comp3 = Vanagon::Component::DSL.new('service-test-3', {}, cur_plat._platform)
             comp3.install_service('SourceDir\\ProgramFilesFolder\\TestID\\TestProduct\\somedir\\oneUpAgain\\twoUpAgain\\bin.exe')
-            expect(cur_plat._platform.generate_service_bin_dirs([comp._component.service, comp2._component.service, comp3._component.service].flatten.compact, proj._project)).to eq( \
-<<-HERE
-<Directory Name="somedir" Id="somedir_0_0">
-<Directory Name="oneUp" Id="oneUp_0_1">
-<Directory Name="twoUp" Id="twoUp_0_2">
-<Directory Id="SERVICETEST1BINDIR" />
-</Directory>
-</Directory>
-<Directory Name="oneUpAgain" Id="oneUpAgain_1_1">
-<Directory Name="twoUp" Id="twoUp_1_2">
-<Directory Id="SERVICETEST2BINDIR" />
-</Directory>
-<Directory Name="twoUpAgain" Id="twoUpAgain_2_2">
-<Directory Id="SERVICETEST3BINDIR" />
-</Directory>
-</Directory>
-</Directory>
-HERE
-            )
+            expect(cur_plat._platform.generate_service_bin_dirs([comp._component.service, comp2._component.service, comp3._component.service].flatten.compact, proj._project))
+              .to eq( \
```

Comment:
@mckern I don't know what the backslash here does, but you removed it in the rest of your cleanup, so not sure if you wanted it here or not.